app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -545,52 +550,52 @@
     <string name="configLabel_themeColorBorder">Borda</string>         <!-- color label -->
     <string name="configHint_themeColorBorder">#FFFFFFFF</string>        <!-- color hint -->
 
-    <string name="configLabel_themeBorderWidthRiseSet">Tamanho da Borda (Nasc/pôr)</string>  <!-- size label -->
+    <string name="configLabel_themeBorderWidthRiseSet">Largura da Borda (nascer/pôr)</string>  <!-- size label -->
     <string name="configHint_themeBorderWidthRiseSet">2</string>                          <!-- size hint -->
-    <string name="configLabel_themeBorderWidthNoon">Tamanho da Borda (tarde)</string>         <!-- size label -->
+    <string name="configLabel_themeBorderWidthNoon">Largura da Borda (tarde)</string>         <!-- size label -->
     <string name="configHint_themeBorderWidthNoon">3</string>                             <!-- size hint -->
 
-    <string name="configLabel_themeColorSuffix">Cor Sufixo</string>   <!-- color label -->
```

Comment:
Is this a reversion?

Reply:
Ditto for "Text Colour2" and "Title Size", "Suffix Size", and "Padding".

I want to make sure these changes back to English are intentional (and not accidental pulled in from the default).